doc/user_guide/forwardSamplingExample.tex

```diff
@@ -51,10 +51,10 @@ \subsection{Monte-Carlo sampling through RAVEN}
 \begin{enumerate}
    \item \textbf{\textit{RunInfo}}:
      \xmlExample{framework/user_guide/ForwardSamplingStrategies/forwardSamplingMontecarlo.xml}{RunInfo}
-     As discussed in Section~\ref{sub:EntitiesAndFlow}, the \textit{RunInfo} \textbf{Entity} sets up the analysis
+     As discussed in Section~\ref{sub:singleRun}, the \textit{RunInfo} \textbf{Entity} sets up the analysis
      that the user wants to perform. In this specific case, two steps (\xmlNode{Sequence}) are sequentially run
-     using 12 processors (\xmlNode{batchSize}). This means that
-     12 instances of the driven code are run simultaneously. Every time a simulation ends, a new one is launched.
+     using 1 processor (\xmlNode{batchSize}). If this number change into 12, this means that
```

Comment:
maybe "If this number were changed to 12"?

Reply:
Changed